release/_publishing.sh

```diff
@@ -42,6 +42,30 @@ function add_fixed_issues_to_project_version {
 	lc_log INFO "The full fixed issues list has been added to the ${_PRODUCT_VERSION} project version"
 }
 
+function add_patcher_project_version {
+	local add_by_name_response=$(\
+		curl \
+			"https://patcher.liferay.com/api/jsonws/osb-patcher-portlet.project_versions/addByName" \
+			--data-raw "combinedBranch=true&committish=${_PRODUCT_VERSION}&fixedIssues=&name=${_PRODUCT_VERSION}&productVersionLabel=Quarterly Releases&repositoryName=liferay-portal-ee&rootPatcherProjectVersionName=" \
```

Comment:
@kiwm, the parameter `productVersionLabel` may vary, e.g. it can be `7.4`, `7.3`, etc

Reply:
My understanding is that this code is intended to add new project versions via an API call. If a project version with the same name already exists, the operation will not work. For U Project versions, some new variables are needed, including the Project Version Root and the Product Version Name. However, I am not sure exactly where to find these variables. The logic for the Fixed Issues current depends on the previous released version don't contens the full list of tickets.